front/gatsby/src/components/Layout/Header/Header.module.css

```diff
@@ -1,5 +1,12 @@
-.header {
+.headerTop {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: space-evenly;
+  align-items: stretch;
```

Comment:
btw, that's the "official" way by Gatsby in their tutorial to add CSS. But a more elegant alternative could be using tachyon classes and removing all these .module.css files

we could have a chat with @goldsteinsveta to decide how we would like the design-system's components to look like, for the end user.

Reply:
SGTM, I'm always up for less CSS.